components/clp-py-utils/clp_py_utils/initialize-clp-metadata-db.py

```diff
@@ -95,6 +95,16 @@ def main(argv):
                 """
             )
 
+            metadata_db_cursor.execute(
+                f"""
+                CREATE TABLE IF NOT EXISTS `{table_prefix}column_metadata_default` (
+                    `name` VARCHAR(512) NOT NULL,
+                    `type` INT NOT NULL,
```

Comment:
We need to make the same change in `indexer`, right?

Reply:
Sure, I will submit a PR later.

Reply:
What if we use TINYINT?

Reply:
I'm fine with TINYINT. Yeah, let's add the necessary binding in MySQLParamBindings.